lib/index.test.js

```diff
@@ -0,0 +1,182 @@
+import fs from 'fs';
+import path from 'path';
+import { rollup } from 'rollup';
+import multiEntry from 'rollup-plugin-multi-entry';
+
+import rootImport from './';
+
+const fixtures = path.resolve(__dirname, '../test/support/fixtures');
+
+jest.mock('fs', () => {
+  const fs = jest.requireActual('fs');
+  jest.spyOn(fs, 'stat');
+  return fs;
+});
+
+function runRollup(input, { plugins = [], ...options } = {}) {
+  return rollup({
+    input,
+    plugins: [rootImport(options), ...plugins],
+    onwarn() {},
+  }).then(async (bundle) => {
+    const { output } = await bundle.generate({
+      format: 'cjs',
+    });
+    expect(output).toHaveLength(1);
+    return output[0].code;
+  });
+}
+
+function runModule(code) {
```

Comment:
This seems like it should be in a standard library for rollup plugin testing

Reply:
Heh yeah